### PR TITLE
Gt styled recipes

### PR DESCRIPTION
**Pull request by Dream-Master**
add(CFB)GT styled recipes

### CODE REVIEW

**Comment by iouter** [comment]
about cooking for blockheads II recipe, i think use the steel plate is better. this allow player use the mod at steam age, not after the lv cutting machine.

**Comment by Dream-Master** [comment]
@iouter changed

**Comment by iouter** [comment]
alright

File: src/main/java/net/blay09/mods/cookingforblockheads/CommonProxy.java [hunk 1]
```diff
@@ -71,50 +71,94 @@ public void init(FMLInitializationEvent event) {
...
+			}
 		}
-
+		
 		// Cooking Oven
-		if(CookingConfig.enableOven) {
-			GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(CookingForBlockheads.blockOven), "GGG", "IFI", "III", 'G', new ItemStack(Blocks.stained_glass, 1, 15), 'I', "ingotIron", 'F', Blocks.furnace));
+		if (CookingConfig.enableOven) {
+			if (Loader.isModLoaded("gregtech")) {
+				GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(CookingForBlockheads.blockOven), "GGG", "IFI", "III", 'G', new ItemStack(Blocks.stained_glass_pane, 1, 15), 'I', "plateIron", 'F', "craftingIronFurnace"));
```
Comment (Prometheus0000) [inline src/main/java/net/blay09/mods/cookingforblockheads/CommonProxy.java:119]: Are stained glass and iron furnaces possible pre-steam? I'm not sure if they are.

File: src/main/java/net/blay09/mods/cookingforblockheads/CommonProxy.java [hunk 2]
```diff
@@ -71,50 +71,94 @@ public void init(FMLInitializationEvent event) {
...
+		} else {
+			GameRegistry.addShapelessRecipe(new ItemStack(CookingForBlockheads.blockFridge), Blocks.chest, Items.iron_door);
+		}
 
 		// Sink
-		if(CookingConfig.enableSink) {
-			GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(CookingForBlockheads.blockSink), "III", "WBW", "WWW", 'I', "ingotIron", 'W', "logWood", 'B', Items.water_bucket));
+		if (CookingConfig.enableSink) {
+			if (Loader.isModLoaded("gregtech")) {
+				GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(CookingForBlockheads.blockSink), "III", "WBW", "WWW", 'I', "plateIron", 'W', "plateWood", 'B', Items.water_bucket));
```
Comment (Prometheus0000) [inline src/main/java/net/blay09/mods/cookingforblockheads/CommonProxy.java:92]: Does this plate need a compressor? If so, that pushes it to steam. I thought it was supposed to be pre-steam?

Same for other platewoods.

File: src/main/java/net/blay09/mods/cookingforblockheads/CommonProxy.java [hunk 3]
```diff
@@ -71,50 +71,94 @@ public void init(FMLInitializationEvent event) {
 		}
 
 		// Cooking for Blockheads II
-		if(CookingConfig.enableCraftingBook) {
-			GameRegistry.addRecipe(new ItemStack(CookingForBlockheads.itemRecipeBook, 1, 1), " C ", "DBD", " C ", 'C', Blocks.crafting_table, 'D', Items.diamond, 'B', CookingForBlockheads.itemRecipeBook);
+		if (CookingConfig.enableCraftingBook) {
+			if (Loader.isModLoaded("gregtech")) {
+				GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(CookingForBlockheads.itemRecipeBook, 1, 1), " C ", "DBD", " C ", 'C', Blocks.crafting_table, 'D', "plateSteel", 'B', CookingForBlockheads.itemRecipeBook));
```
Comment (Prometheus0000) [inline src/main/java/net/blay09/mods/cookingforblockheads/CommonProxy.java:76]: I thought it was supposed to be pre-steam? Steel is in the middle of steam.

File: src/main/java/net/blay09/mods/cookingforblockheads/CommonProxy.java (hunk 1 above)
Comment (Dream-Master) [reply src/main/java/net/blay09/mods/cookingforblockheads/CommonProxy.java:119]: why it must be avaialable pre steam ? Yea have much todo with other stuff. Planting stuff is always steam age in my playthrough

File: src/main/java/net/blay09/mods/cookingforblockheads/CommonProxy.java (hunk 3 above)
Comment (Dream-Master) [reply src/main/java/net/blay09/mods/cookingforblockheads/CommonProxy.java:76]: why it must be avaialable pre steam ? Yea have much todo with other stuff. Planting stuff is always steam age in my playthrough

File: src/main/java/net/blay09/mods/cookingforblockheads/CommonProxy.java (hunk 1 above)
Comment (Dream-Master) [reply src/main/java/net/blay09/mods/cookingforblockheads/CommonProxy.java:119]: thanks for your review but lets hear mitch on this as well. And as a pack creator i guess my ideas and words counts too?

File: src/main/java/net/blay09/mods/cookingforblockheads/CommonProxy.java (hunk 2 above)
Comment (mitchej123) [reply src/main/java/net/blay09/mods/cookingforblockheads/CommonProxy.java:92]: I'm fine with steam.  Iron/wood/steel plates are fine.  I think it was already removed, but I'd avoid Diamond plates or anyting.

File: src/main/java/net/blay09/mods/cookingforblockheads/CommonProxy.java (hunk 3 above)
Comment (mitchej123) [reply src/main/java/net/blay09/mods/cookingforblockheads/CommonProxy.java:76]: I want it early game.  Steam age is fine, or some pieces LV.

**Comment by Prometheus0000** [comment]
I'm fine with steam, I just wasn't sure you wanted it there. Make sure to move the quest from pre-steam to steam in an appropriate place (after steel?) when you finish with the recipes, though.

**Comment by Dream-Master** [comment]
Sure will. Didn't know you do quests already.

**Comment by mitchej123** [comment]
Ideally we'd cache the GT enabled instead of querying it every time, but it's at load time, and probably isn't *that* expensive, so it should be fine for now.  As for the recipes as long as it's Steam/LV I'm fine with it.

**Review by mitchej123 — APPROVED** [review]

**Comment by Prometheus0000** [comment]
Actually now that I think of it I think I moved it to the cooking farming tab, with a pre-req of iron. Changing it to steel pre-req should be sufficient unless I'm remembering wrong again.

**Comment by iouter** [comment]
> Actually now that I think of it I think I moved it to the cooking farming tab, with a pre-req of iron. Changing it to steel pre-req should be sufficient unless I'm remembering wrong again.

i think it’s not difficult to find 3 or 4 steel ingots

**Pull request merged**